trio/_core/_io_windows.py

```diff
@@ -245,9 +244,9 @@ class CKeys(enum.IntEnum):
 # operation and start a new one.
 @attr.s(slots=True, eq=False)
 class AFDWaiters:
-    read_task: Optional[_core.Task] = attr.ib(default=None)
-    write_task: Optional[_core.Task] = attr.ib(default=None)
-    current_op: Optional[AFDPollOp] = attr.ib(default=None)
+    read_task: _core.Task | None = attr.ib(default=None)
+    write_task: _core.Task | None = attr.ib(default=None)
+    current_op: AFDPollOp | None = attr.ib(default=None)
```

Comment:
Eh, same as above. I won't make any more comments about 3.9+ features sneaking in.